src/main/java/mate/academy/spring/dto/request/MovieRequestDto.java

```diff
@@ -1,6 +1,9 @@
 package mate.academy.spring.dto.request;
 
+import javax.validation.constraints.NotNull;
+
 public class MovieRequestDto {
+    @NotNull
```

Comment:
what if title will is blank?

Reply:
I don't know any movie without a title)  But I deleted it for our task

Reply:
I mean you are validating DTO when you registering the movie. You don't know any movie without a title. So do I. But this code will make it possible to create a movie with title = ""